content/developer/reference/backend/testing.rst

```diff
@@ -540,8 +540,131 @@ business flow.  It explains a sequence of steps that should be followed.  The
 test runner will then create a Chrome headless browser, point it to the proper url
 and simulate the click and inputs, according to the scenario.
 
+Writing a test tour
+-------------------
+
+Structure
+~~~~~~~~~
+
+.. code-block:: text
+
+    your_module
+    `-- ...
+    `-- static
+        `-- tests
+            `-- tours
+                |-- your_tour.js
+    `-- tests
+        |-- __init__.py
+        |-- test_calling_the_tour.py
+    |-- __manifest__.py
+
+and ``__manifest__.py`` contains:
+
+.. code-block:: python
+
+    'assets': {
+        'web.assets_tests': [
+            'your_module/static/tests/tours/your_tour.js',
+        ],
+    },
+
+and as for any python test, ``__init__.py`` in the folder ``tests`` contains::
+
+    from . import test_calling_the_tour
+
+Javascript
+~~~~~~~~~~
+
+1. Setup your tour by defining it
+
+    .. code-block:: javascript
+
+        odoo.define('rental_product_configurator_tour', function (require) {
+        "use strict";
+
+        let tour = require('web_tour.tour');
+
+        tour.register('rental_product_configurator_tour', {
+            url: "/web",
+            test: true,
+        }, [tour.stepUtils.showAppsMenuItem(), {
+            // First step. Here, starting from the Apps Menu
+            trigger: '.o_app[data-menu-xmlid="sale_renting.rental_menu_root"]',
+            edition: 'enterprise'  // You don't have to precise your edition if it's in community
+        }, {
+            // Another tour step
+        },
+        ]);
+        });
+
+2. Add any step you want, for examples:
+
+.. code-block:: javascript
+
+    {
+        trigger: '.js_product:has(strong:contains(Chair floor protection)) .js_add',
+        extra_trigger: '.oe_advanced_configurator_modal',  // This ensure we are in the wizard
+        run: 'click'
+    },
+
+.. code-block:: javascript
+
+    {
+        trigger: "a:contains('Add a product')",
+        // Extra-trigger to make sure a line is added before trying to add another one
+        extra_trigger: '.o_field_many2one[name="product_template_id"] .o_external_button',
+    },
+
+Here are some possible arguments for your steps:
+
+- **trigger**: element/class/selector of the DOM you want to trigger
```

Comment:
`selector/Element/jQuery` (see the documentation of [find](https://api.jquery.com/find/), feel free to link that documentation here too)

"class" is incorrect as a class alone without a `.` will be interpreted as a tag selector. Avoid using "DOM" in this context, as it is not accurate. Same comment applies to the extra_trigger